Automattic/wp-calypso
# Site Settings: removes After the Deadline

#### Changes proposed in this Pull Request

This PR removes all After the Deadline references from the Site's Settings _Writting_ tab for Jetpack sites.

| Before | After |
| - | - |
| ![Screen Shot 2019-03-29 at 18 55 27](https://user-images.githubusercontent.com/233601/55264860-98744d00-5254-11e9-8acf-b41807f5e828.png) | ![Screen Shot 2019-03-29 at 18 52 36](https://user-images.githubusercontent.com/233601/55264869-a1651e80-5254-11e9-9fb5-9db6d4c8f9e4.png) |


#### Testing instructions



On a Jetpack site, verify that the remaining _Writting_ > _Composing_ settings work as expected and there are no regressions.

Part of #28550, and complements of https://github.com/Automattic/jetpack/pull/11772
